examples/reference/elements/matplotlib/HLine.ipynb

```diff
@@ -37,10 +37,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts HLine (color='blue' linewidth=6) Points (color='#D3D3D3')\n",
     "xs = np.random.normal(size=100)\n",
     "ys = np.random.normal(size=100) * xs\n",
-    "hv.Points((xs,ys)) * hv.HLine(ys.mean())"
+    "points = hv.Points((xs,ys))\n",
+    "\n",
+    "points.opts(color='blue', linewidth=6) * hv.HLine(ys.mean()).opts(color='#D3D3D3')"
```

Comment:
Another case where I would do:

```python
hline = hv.HLine(ys.mean())
points.opts(color='blue', linewidth=6) * hline.opts(color='#D3D3D3')
```

But would be okay but wouldn't love doing:

```python
hline = 
(points * hv.HLine(ys.mean())).opts(
    opts.HLine(color='#D3D3D3')
    opts.Points(color='blue', linewidth=6))
```

Reply:
Again, I prefer the latter!